larky/src/test/resources/test_loading_module.star

```diff
@@ -5,6 +5,7 @@ load("@stdlib//json", "json")
 load("@stdlib//hashlib", "hashlib")
 load("@vendor//Crypto/Hash", "Hash")
 load("testlib/builtinz", "setz", "collections")
+load("github.com/verygoodsecurity/larky-helpers", "mirror")
```

Comment:
This is not right. We should use bazel's concept of loading files from a repo.

Isn't this the same thing as using something like bzlmod?

https://bazel.build/docs/external

Reply:
https://github.com/cirrus-modules/helpers#example

i was using this as an example. here it's loading the dependency from a third party repository.

we could load them the way you're describing but it may be less obvious to a user how they would bring in third party dependencies. the github.com/... syntax is obvious to understand imo

Reply:
This breaks hermetic seal properties. What if gh is down? Then we no longer have hermetic execution or deterministic properties.

Reply:
bzlmod acts like requirements.txt which would be just as familiar. Especially if it's a common repo that's continuously cached

Reply:
yes, we cannot load this at runtime, that makes sense. will look at this...